pkg/sentry/platform/kvm/machine_unsafe.go

```diff
@@ -143,3 +119,55 @@ func (c *vCPU) waitUntilNot(state uint32) {
 		panic("futex wait error")
 	}
 }
+
+// setUserRegisters sets user registers in the vCPU.
+func (c *vCPU) setUserRegisters(uregs *userRegs) error {
+	if _, _, errno := syscall.RawSyscall(
+		syscall.SYS_IOCTL,
+		uintptr(c.fd),
+		_KVM_SET_REGS,
```

Comment:
KVM_SET_REGS and KVM_GET_REGS were not supported on arm64 which only support KVM_SET_ONE_REG.

Reply:
getUserRegisters is arch-specific(no KVM_SET_REGS/KVM_GET_REGS on Arm64), so this function can't be called in bluepill.go:die(), otherwise an throw will occuer.
The api of getUserRegisters  should be moved into bluepill_amd64_unsafe.go:dieArchSetup()

Reply:
ok,ok,ok. I see this is totally wrong. I have closed this pr. Could one of you create a proper fix for this? I want to enable travis-CI, but right now this code can't be compiled for arm64.

If you look at my travis-ci branch, you can find that I had to do a few hacks what we need to fix properly:
https://github.com/avagin/gvisor/commits/arm64-travis2